pkg/http-server/file-scan.go

```diff
@@ -35,6 +35,7 @@ import (
 
 // scanFile accepts uploaded file and runs scan on it
 func (g *APIHandler) scanFile(w http.ResponseWriter, r *http.Request) {
+	zap.S().Info("handle: file scan request")
```

Comment:
It is my suggestion to start using zap loggers rather than using global loggers, in this case we can initialize the logger with some params like:
```Go
logger := logging.GetDefaultLogger().With(zap.String("endpoint", "/v1/file/scan"))
```

Reply:
I agree, can we maintain a singleton logger object for this purpose?

Reply:
These changes can be made while working on logging middleware.